### ISSUE TITLE
Windows Defender classifies lychee as malware 

### ISSUE THREAD

**Issue by Aspie96** [lycheeverse/lychee#1250]
Windows defender detects lychee as a malware.

**Comment by mre** [comment]
Thanks. I'm not sure what to do about that. Is there a way to reach out and ask them to look into that?

**Comment by mre** [comment]
I tried to submit a false-positive report on the [Windows Defender Security Intelligence Submission page](https://www.microsoft.com/en-us/wdsi/filesubmission).

For that, I need some more info.

* Which Windows version do you use?
* I need to specify the Detection name. It points me to these instructions, which gets the detection name from the [history page of detected threats on Microsoft Defender Antivirus](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-security-center-antivirus?view=o365-worldwide#review-threat-detection-history-in-the-windows-defender-for-cloud-app).

Could you help me with that?

**Comment by Aspie96** [comment]
I use Windows 10 Home, version 22H2 and the file I tried to download is `lychee-nightly-windows-x86_64.exe`.

Yesterday the file was first [blocked by Google Chrome](https://support.google.com/chrome/answer/6261569) then, upon confirming the download, it was immediately deleted by Windows Defender, which deemed the threat to he "High". I am not sure what "detection name" refers to, however clicking on "further information" links [here](https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Program%3AWin32%2FWacapew.C!ml&threatid=265744).

Today (and I see the file has been updated 19 hours ago) the file is still blocked by Google Chrome, but it's not deleted by Windows Defender, or even detected as a threat if manually scanned.

**Comment by mre** [comment]
Thanks for the additional info.

That is very strange. I doubt that Windows Defender changes their detection algorithm that frequently. Then again, maybe they flag everything by default and have some tracking/telemetry to run new binaries through a pipeline. 🤔 

[Chrome has their own set of rules for treating downloads.](https://support.google.com/chrome/a/answer/7579271?hl=en) Not sure if there's much we can do there either.
For what it's worth, I just downloaded the latest Windows binary from Chrome on macOS and it worked. Then I tried it in the latest Chrome on a Windows 10 virtual machine, and it worked as well: no download issues and the binary works as expected.

I'd say we can close the issue as it seems to work again. If we run into more issues into the future, we can reopen it.
